Dockerfile

```diff
@@ -19,9 +21,12 @@ RUN cargo build --release $CARGO_OPTS && \
     strip target/release/websocat
 
 # Final stage
-FROM alpine:3.18
+FROM debian:bookworm
```

Comment:
Why debian for the final stage as well?

Maybe better turn `crt-static` back on and use empty (`scratch`) container as a base?

---

Why does dockerized Websocket even needed in the first place? As a stand-alone container or as a basis for other containers?

Reply:
glibc doesn't do fully static AFAIU. So we need an image with glibc. There is a bookworm-slim image but couldn't install libssl and didn't investigate. I guess if just static works, then bookworm-slim can be used to save 10MB.

My use case is just to run websocat on hosts where distro doesn't come with a good version. Or where I don't want or can't to install packages.

If the project publishes static binaries, the container builds can just pick them up instead of compiling under emulation. That would probably be ideal. But probably has to be based on musl or something else non-glibc.

I can support this with some setup but I don't have time right now to investigate how rust builds works, not familiar with the language.